.gitignore

```diff
@@ -16,3 +16,5 @@ lib64
 
 # python virtual environments for testing
 venv/
+
+_build/
```

Comment:
oh wait, there's extra lines now. ah w/e, I'll fix it up if I'm really bothered later. Not a blocker